src/Features/CSharp/Portable/RemoveUnnecessaryImports/AbstractCSharpRemoveUnnecessaryImportsService.Rewriter.cs

```diff
@@ -158,7 +158,24 @@ public override SyntaxNode VisitNamespaceDeclaration(NamespaceDeclarationSyntax
 
             private static SyntaxToken StripNewLines(SyntaxToken firstToken)
             {
-                return firstToken.WithLeadingTrivia(firstToken.LeadingTrivia.SkipWhile(t => t.Kind() == SyntaxKind.EndOfLineTrivia));
+                var trimmedLeadingTrivia = new List<SyntaxTrivia>();
+                foreach(var token in firstToken.LeadingTrivia)
+                {
+                    if(token.Kind() != SyntaxKind.EndOfLineTrivia)
```

Comment:
none of these cases are clear as to what they're doing.

Reply:
I went with your suggestion above - makes it much clearer. thanks :)